section/algorithms.html

```diff
@@ -119,6 +119,50 @@ <h3>Outgoing payment and quote grant request</h3>
             <li>
                 If |authServer| [=url/scheme=] is not `https`, return failure.
             </li>
+            <li>
+                <p>Construct a {{ GrantRequest }} |grantRequest| as follows:</p>
+                <ol>
+                    <li>
+                        Set |grantRequest|'s {{GrantRequest/client}} to <var>walletAddressDetails</var>'s {{WalletAddressDetails/url}} property.
+                    </li>
+                    <li>
+                        Set |grantRequest|'s {{GrantRequest/interact}} to a new {{ Interact }} with {{Interact/start}} set to `["redirect"]`.
+                    </li>
+                    <li>
+                        Let |quoteAccess| be a new {{ Access }} object with:
+                        <ul>
+                            <li>{{Access/type}} set to `"quote"`</li>
+                            <li>{{Access/actions}} set to `["create"]`</li>
+                        </ul>
+                    </li>
+                    <li>
+                        Let |debitAmount| be a new {{ Amount }} object with:
+                        <ul>
+                            <li>{{Amount/value}} set to |budget|</li>
+                            <li>{{Amount/assetScale}} set to <var>walletAddressDetails</var>'s {{WalletAddressDetails/assetScale}} property</li>
+                            <li>{{Amount/assetCode}} set to <var>walletAddressDetails</var>'s {{WalletAddressDetails/assetCode}} property</li>
+                        </ul>
+                    </li>
+                    <li>
+                        Let |accessLimits| be a new {{ AccessLimits }} object with {{AccessLimits/debitAmount}} set to |debitAmount|.
```

Comment:
We likely want to support recurring grant here? The `accessLimits` will have an `interval` property set if so.

I notice we've `interval` as variable available to this section and call-site:
> Let interval be an [ISO 8601 repeating interval](https://en.wikipedia.org/wiki/ISO_8601#Repeating_intervals) if renewMonthly is true. The starting date for the repeating interval needs to be the users current date (ISO String).